examples/web/static/multidownload.html

```diff
@@ -0,0 +1,583 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Dataset Testing </title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link href="main.css" rel="stylesheet">
+
+    <!--    <script src="https://cdn.jsdelivr.net/npm/@eyepop.ai/eyepop/dist/eyepop.min.js"></script>-->
+    <script src="/src/eyepop-data/dist/eyepopdata.min.js"></script>
+    <script src="/src/eyepop-render-2d/dist/eyepop.render2d.min.js"></script>
+
+</head>
+
+<body>
+    <div class="jumbotron vertical-center">
+        <div class="container-fluid mt-4">
+            <div class="row" style="width: 100%">
+                <div class="col text-center" id="list-datasets">
+                    <h1 class="display-4">List Datasets</h1>
+                    <button class="btn btn-primary" id="new-datasets-btn">+ Dataset</button><br /><br />
+                    <div id="list-datasets-result"></div>
+                </div>
+            
+                <div class="col text-center" id="list-assets">
+                    <h1 class="display-4">Asset Viewer</h1>
+                    <h2 class="display-6" id="list-assets-model-info"></h2>
+                    <div id="list-assets-result"></div>
+                </div>
+            </div>
+            <div class="row" style="width: 100%">
+            
+                <div class="col text-center" id="list-models">
+                    <h1 class="display-4">Model Viewer</h1>
+                    <div id="list-models-result"></div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <script>
+
+        const apiKey = 'AAGcsWj8N2PlKQl9c9ydz3QFZ0FBQUFBQm1mZDB5eDUwalNlYi12NWotd3hsVGJiMW1sVXF1dE9aOU9oSGVBOWtBQXoxZmNjUE5Nb1YzY3RROUdzbVUwUkZtcDhZcG5vSWROTzR1TU8ybGhZckx6RTgzYVZwMjZEREZjalZubnpYaUNMWVdBODg9';
+        const auth = apiKey ? { secretKey: apiKey } : { oAuth2: true };
+       
+        const account_uuid = "49326f2e085a46c39ba73f91c52e436c";
```

Comment:
treat account ids as weak secrets too. They should not be exposed if not needed.